packages/api/src/platforms/vtex/resolvers/searchResult.ts

```diff
@@ -38,21 +38,26 @@ export const StoreSearchResult: Record<string, Resolver<Root>> = {
       }
     }
 
-    const { productSearchPromise } = root
     const [terms, productSearchResult] = await Promise.all([
       search.suggestedTerms(searchArgs),
       productSearchPromise,
     ])
 
-    const skus = productSearchResult.products
-      .map((product) => {
-        // What determines the presentation of the SKU is the price order
-        // https://help.vtex.com/pt/tutorial/ordenando-imagens-na-vitrine-e-na-pagina-de-produto--tutorials_278
-        const maybeSku = pickBestSku(product.items)
-
-        return maybeSku && enhanceSku(maybeSku, product)
-      })
-      .filter((sku) => !!sku)
+    const skus = await Promise.all(
+      productSearchResult.products
+        .map((product) => {
+          // What determines the presentation of the SKU is the price order
+          // https://help.vtex.com/pt/tutorial/ordenando-imagens-na-vitrine-e-na-pagina-de-produto--tutorials_278
+          const maybeSku = pickBestSku(product.items)
+
+          return maybeSku && enhanceSku(maybeSku, product)
+        })
+        .filter((sku) => !!sku)
+        .map(async (sku) => ({
+          ...sku,
+          rating: await commerce.rating(sku.itemId),
+        }))
```

Comment:
🪂  This is not good for performance (latency) I'll explain more in the next instance above.